Site/_sass/extras-styles.scss

```diff
@@ -227,22 +227,39 @@
 
 
 /***** DOC LAYOUT *****/
+@import 'sidebar-collapse';
+
 .light header {
   box-shadow: 0 0 25px 0 rgba(0, 0, 0, 0.1);
 }
 
+#sidebar-collapse {
+  cursor: pointer;
+  @include sidebar-collapse-transition(transform);
+}
+
```

Comment:
Currently, the icon is reversed, reflecting the current state instead of the intended action. I suggest adding a transformation here or changing the SVG being used to address this issue.

Reply:
Done!